node/pkg/node/adminServiceRunnable.go

```diff
@@ -25,7 +25,7 @@ import (
 	ethcrypto "github.com/ethereum/go-ethereum/crypto"
 )
 
-func AdminServiceRunnable(
+func adminServiceRunnable(
```

Comment:
If you are going to change this name, shouldn't it be consistent with the naming used in `node/pkg/node/guardianKey.go`?

Reply:
I thought that `GuardianKeyToAddress` may be useful in other places of the codebase and it'd be a good function to export vs. `adminServiceRunnable`, which isn't and shouldn't be used elsewhere. Wdyt?

Reply:
Wasn't aware of the Go naming convention stuff.  That should be fine.